### PR TITLE
Optimization code

### LINKED ISSUE #26: Optimize code in hidden links
Optimize code by replacing the jquery code used to temporarily hide agenda links generated by the controller expedient by code that removes the link in a correct way.

### LINKED ISSUE #27: Optimize code replacing datetime with request.now to avoid problems with diferents time zones

### PR DESCRIPTION
**Pull request by Juerodriguez**
## Resumen

close https://github.com/Juerodriguez/OpenLex/issues/26
close https://github.com/Juerodriguez/OpenLex/issues/27

Optimize code that had been temporarily solved with jquery
Optimize code replacing datetime with request.now to avoid problems with different time zones

## Checklist

- [x] Variables, funciones y comentarios en Español
- [x] Corrector ortográfico ok
- [x] Cdigo Python cumple PEP8
- [x] Cobertura de tests con Playwright
- [x] Revisores asignados (pares & al menos 1 mentor)

## Screenshots

(vista rápida, preferir png, gif animado y/o video de Playwright)

### CODE REVIEW

File: controllers/dashboard.py [hunk 1]
```diff
@@ -28,12 +26,12 @@ def view():
         exportclasses=myexport)
     query_agenda = (db.agenda.created_by == auth.user.id)
     query_agenda &= (db.agenda.estado == 'P')
-    ahora = datetime.datetime.now()
+    ahora = request.now
```
Comment (eamanu) [inline controllers/dashboard.py:29]: good catch

File: controllers/expedientes.py [hunk 2]
```diff
@@ -125,6 +125,7 @@ def convert_html_to_pdf(source_html, output_filename):
 
 def vista_expediente():
     'muestra un panel a la izquierda que tiene los datos del expediente y permite navegar en él'
+    LINKED_TABLES.append("agenda")
```
Comment (eamanu) [inline controllers/expedientes.py:128]: Guarda con esto, al ser un append() si lo ejecutas varias veces vas a empezar a tener "agenda" agregada varias veces a `LIKED_TABLES`, por favor chequearlo.

File: controllers/dashboard.py [hunk 3]
```diff
@@ -6,8 +6,6 @@
 __author__ = "María Andrea Vignau (mavignau@gmail.com)"
 __copyright__ = "(C) 2016 María Andrea Vignau. GNU GPL 3."
```
Comment (Juerodriguez) [inline controllers/dashboard.py:8]:
```suggestion
import datetime
```

**Review by NicolasSandoval — APPROVED** [review]
Great job! Thank you.

File: controllers/expedientes.py (hunk 2 above)
Comment (eamanu) [reply controllers/expedientes.py:128]: @NicolasSandoval @Juerodriguez

**Pull request merged**